antora.yml

```diff
@@ -11,6 +11,8 @@ asciidoc:
     # used for things like rpk download URLs
     full-version: 24.1.4
     latest-release-commit: 'b5ade3f40'
+    # RP Connect latest tagged release
+    tagged-version: 4.30.1 
```

Comment:
I've added an attribute for tagged releases so that the paths for the released zip files stay up to date.